groups/dps/main.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">= 1.3"
+
+  backend "s3" {}
+
+  required_providers {
+    aws = {
+      version = ">= 5.0, < 6.0"
+      source  = "hashicorp/aws"
+    }
+  }
```

Comment:
We should explicitly define the `vault` and `cloudinit` providers here for completeness.

E.g.:
```
  required_providers {
    aws = {
      version = ">= 5.0, < 6.0"
      source  = "hashicorp/aws"
    }
    cloudinit = {
      source  = "hashicorp/cloudinit"
      version = ">= 2.3.0, < 3.0.0"
    }
    vault = {
      source  = "hashicorp/vault"
      version = ">= 3.23.0, < 4.0.0"
    }
  }
```